src/sentry/db/models/fields/node.py

```diff
@@ -156,7 +156,7 @@ def save(self, subkeys=None):
         subkeys = subkeys or {}
         subkeys[None] = to_write
 
-        nodestore.backend.set_subkeys(self.id, subkeys)
+        nodestore.backend.set_subkeys(self.id, subkeys, override_cache=True)
```

Comment:
This write to cache makes sense to me. The node could be in cache (because of a read) and any update to the data should also reflect in future reads which could be served from cache.